Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -163,8 +164,38 @@ namespace OpenInfraPlatform
 						dAngle += M_TWOPI;
 					}
 				}
+				/*!Compare two doubles for equality to the specified precision
+				*\param[in]    first      The first value
+				*\param[in]    second     The second value
+				*\param[in]    precision  Precision of model
+				
+				*\return true, if the absolute difference is smaller than the precision. False otherwise
+				*/
+				bool areEqual(const double first,const double second,const double precision) const
+				{
+					return abs(first - second) < precision;
+				}
+				/*!Compare two 2D vectors for equality to the specified precision
+				*\param[in]    first      The first 2D-point of vector
+				*\param[in]    second     The second 2D-point of vector
 
+				*\return true, if the absolute difference is smaller then the precision; otherwise false
+				*/
+				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
+				{
+					return areEqual(first.x, second.x, getPrecision()) && areEqual(first.y, second.y, getPrecision());
+				}
+				/*!Compare two 3D vectors for equality to the specified precision
+				*\param[in]    first      The first 3D-point of vector
+				*\param[in]    second     The second 3D-point of vector
 
+				*\return true, if the absolute difference is smaller then the precision; otherwise false

```

Comment:
Please correct this line, like you have already done in `line 172`.

Reply:
Resolved: 7305a03